tcs-persistence/src/main/java/com/transformuk/hee/tis/tcs/service/model/ConditionsOfJoining.java

```diff
@@ -15,12 +20,17 @@
  */
 @Data
 @Entity
-public class ConditionsOfJoining {
+public class ConditionsOfJoining implements Serializable {
 
   @Id
   @Type(type = "org.hibernate.type.UUIDCharType")
   private UUID programmeMembershipUuid;
   private Instant signedAt;
   @Enumerated(EnumType.STRING)
   private GoldGuideVersion version;
+
+  @OneToOne(mappedBy = "conditionsOfJoining", cascade = CascadeType.REMOVE)
+  @PrimaryKeyJoinColumn(name = "programmeMembershipUuid")
+  @JoinColumn(name = "programmeMembershipUuid")
+  private ProgrammeMembership programmeMembership;
```

Comment:
`programmeMembershipUuid` above looks like a duplicate field now.

Reply:
It's a bit weird because it's both the foreign key but also the PK.